includes/REST_API/Stories_Controller.php

```diff
@@ -320,11 +320,37 @@ public function get_items( $request ) {
 			$posts_query->query( $query_args );
 			$statuses_count[ $key ] = absint( $posts_query->found_posts );
 		}
+
+		if ( $request['story_format'] ) {
```

Comment:
I don't understand why you're doing this whole `story_format` thing whereas the `X-WP-TotalPages` header works just fine and `rest_preload_api_request` passes all headers along.